src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,13 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => {
+  return (
+    <div className="CommentList">
+      {comments.map(comment => (
+        <div id={comment.id}>
+          <CommentInfo comment={comment} />
+        </div>
+      ))}
```

Comment:
Each element in the list should have a unique `key` prop to help React identify which items have changed, are added, or are removed. Consider adding `key={comment.id}` to the `<div>` element.